core/src/test/java/com/linecorp/armeria/server/VirtualHostBuilderTest.java

```diff
@@ -327,4 +329,17 @@ void precedenceOfDuplicateRoute() {
         final Route route = serviceConfig.route();
         assertThat(route).isSameAs(routeA);
     }
+
+    @Test
+    void multipartUploadsLocationCustomization() {
+        Path multipartUploadsLocation = FileSystems.getDefault().getPath("logs", "access.log");
+        final VirtualHost h1 = new VirtualHostBuilder(Server.builder(), false)
```

Comment:
To be honest, I don't agree with the idea because this file is written to test behavior of `VirtualHostBuilder` not `ServerBuilder`.
If we really want to test with `ServerBuilder` we can do that with `ServerBuilderTest`.
How do you think?

Reply:
I agree with your opinion. The server, service and virtual host logic has a complex hierarchy and dependency. So I wanted to check if the value set to a custom virtual host is correctly returned by the getter. It seems overkill.